finetuner/tuner/evaluation.py

```diff
@@ -38,6 +38,10 @@ def __init__(
         eval_data: 'DocumentSequence',
         catalog: 'DocumentSequence',
         embed_model: Optional['AnyDNN'] = None,
+        device: str = 'cpu',
+        batch_size: int = 256,
+        preprocess_fn: Optional['PreprocFnType'] = None,
+        collate_fn: Optional['CollateFnType'] = None,
         distance: str = 'cosine',
         limit: int = 20,
```

Comment:
limit should be right after `embed_model`. Reasoning: `limit` is a logical parameter, while the others are "technical".